### PR TITLE
Testing a pull request

### PR DESCRIPTION
**Pull request by evelina89**
added some content to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,8 @@
 	<p>adding some code</p>
 
 	<p>gg wp</p>
+	
+	<p>more content</p>
```
Comment (ch89) [inline index.html]: This needs to be fixed

Comment (ch89) [inline index.html]: test

File: index.html [hunk 2]
```diff
@@ -12,6 +12,8 @@
...
```
Comment (ch89) [inline index.html]: seems nice

File: index.html (hunk 1 above)
Comment (ch89) [inline index.html]: seems nice

Comment (evelina89) [inline index.html]: test

**Comment by ch89** [comment]
thank you!

**Pull request merged**